doc/deps.md

```diff
@@ -53,40 +55,52 @@ sudo apt-get install byacc flex
 
 tar -xvf ./thrift-<version>.tar.gz
 cd thrift-<version>
-./configure --prefix=<thrift_install_dir>
+
+./configure --prefix=<thrift_install_dir> --with-python=NO --with-php=NO
 # Thrift can generate stubs for many programming languages. The configure script
 # looks at the development environment and if it finds the environment for a
 # given language then it'll use it. For example in the previous step npm was
 # installed which requires NodeJS. If NodeJS can be found on your machine then
 # the corresponding stub will also compile. If you don't need it then you can
 # turn it off: ./configure --witout-nodejs.
+#
+# In certain cases, installation may fail if development libraries for languages
+# are not installed on the target machine. E.g. if Python is installed but the
+# Python development headers are not, Thrift will unable to install.
+# Python, PHP and such other Thrift builds are NOT required by CodeCompass.
+
 make install
 ```
 
 ## Known issues
 - In Ubuntu 16.04 LTS the LLVM/Clang has some packaging issues, i.e. some libs
   are searched in `/usr/lib` however the package has these in
-  `/usr/lib/llvm-3.8/lib` (see http://stackoverflow.com/questions/38171543/error-when-using-cmake-with-llvm).
+  `/usr/lib/llvm-3.8/lib` (see
```

Comment:
LLVM 3.8 related packaging issues are not relevant any more.

Reply:
They are relevant in the sense that there is no indication that the packaging will change once LLVM 6.0's release version hits the upstream repositories, if they do.